subprojects/logging/src/main/java/org/gradle/internal/logging/sink/PlainConsoleDispatchingListener.java

```diff
@@ -42,9 +44,13 @@ public void onOutput(OutputEvent event) {
             // TODO - should attempt to flush the output stream prior to writing to the error stream
```

Comment:
We should think about doing this or perhaps just add an issue to the backlog to make this better